{
  long result=0;
  try {
    result=saveInternal(entryPoint,changes,true);
  }
 catch (  Exception e) {
    LOG.error(e);
    return false;
  }
  return true;
}
